apps/cookbook/src/app/component-overview/component-configuration.json

```
@@ -122,43 +122,43 @@
     },
     {
       "title": "Icons",
-      "paragraph": "Kirbys own icon collection which is constantly growing.",
+      "paragraph": "The concept of the Kirby icon is to be rounded and friendly. They help to explain the action, function, etc.",
       "svgPath": "./assets/component-svg/Icons-Kirby-Component-Overview.svg",
       "route": "/home/showcase/icon"
     },
     {
       "title": "Item",
-      "paragraph": "The Kirby-item component is a layout which consists of three possible columns.",
+      "paragraph": "Item is a placeholder that can contain text, icons, avatars, checkbox, buttons etc. It’s a component that structures other components.",
```

```suggestion
      "paragraph": "Item is a placeholder that can contain text, icons, avatars, checkboxes, buttons etc. It’s a component that structures other components.",
```

Since everything else is plural, then I think the same should apply to 'checkbox' :)